packages/hydrogen-integration/manifest.yml

```diff
@@ -0,0 +1 @@
+name: hydrogen-integration
```

Comment:
Question 1: shouldn't this be `@netlify/hydrogen-integration` ?
Question 2: IIRC it was mentioned that build plugin is only needed due to some hardcoded values in hydrogen and in one of recent commits it was removed and later that was reverted back so it still is here - so do we still actually need it? Did something else came up that we need it for other than previously mentioned some hardcoded paths?